### PR TITLE
Lunch bytes April 1st

### PR DESCRIPTION
**Pull request by MILeach**
Provisional information for the LunchBytes GPU talks

### CODE REVIEW

**Review by bobturneruk — APPROVED** [review]
Great work!

File: _events/lunchbytes-2021-04-01.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+date: 2021-04-01
+published: True
+from: "14:00"
+to: "15:00"
+location: "Google Meet"
+speaker: "Freddie Witherden, James Knight, Tom Deakin, Seb James; Host: Matthew Leach (RSE Sheffield)"
+institute:
+title: "Lunch bytes #5: Making GPU Programming More Portable"
+image:
+redirect_from:
```
Comment (willfurnass) [inline _events/lunchbytes-2021-04-01.md]: Do you definitely need this redirect?

File: _events/lunchbytes-2021-04-01.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+category: lunchbytes
+date: 2021-04-01
+published: True
+from: "14:00"
+to: "15:00"
+location: "Google Meet"
+speaker: "Freddie Witherden, James Knight, Tom Deakin, Seb James; Host: Matthew Leach (RSE Sheffield)"
```
Comment (willfurnass) [inline _events/lunchbytes-2021-04-01.md]: Seb James?

**Review by willfurnass — COMMENTED** [review]
Made two minor comments but otherwise LGTM!  

Do should if you've got any questions re the curation/admin processes.

File: _events/lunchbytes-2021-04-01.md (hunk 2 above)
Comment (MILeach) [reply _events/lunchbytes-2021-04-01.md]: Good spot, I used the previous one as a template and missed removing this.

**Review by willfurnass — APPROVED** [review]
Thanks

**Pull request merged**